ops/charm.py

```diff
@@ -488,6 +487,19 @@ def from_yaml(
         return cls(meta, raw_actions)
 
 
+class RelationRole(enum.Enum):
+    peer = 'peer'
+    requires = 'requires'
+    provides = 'provides'
+
+    def is_peer(self) -> bool:
+        """Return whether the current role is peer.
+
+        A convenience to avoid having to import charm.
+        """
+        return self is RelationRole.peer
```

Comment:
'self is' seems like a very strange syntax. Is it actually guaranteed that every instance of a role is exactly one?
I've proven well enough to myself that it is:
```
>>> r = RelationRole('%s%s'%('pe', 'er'))
>>> r.is_peer()
True
```